article/articles.py

```diff
@@ -21,10 +21,10 @@ def __init__(self, uuid, title, keywords, tasks,
         self.parent = parent
 
     @cached_property
-    def time_to_read(self):
+    def time_to_view(self):
```

Comment:
don't love the name, but understand the confusion with 'read' past tense :-)

Reply:
as discussed, let's keep it as `time_to_read` while changing ArticleRead to ArticleViewed